test/context/yaml/context.test.js

```diff
@@ -541,23 +541,18 @@ describe('#YAML context validation', () => {
     cleanThenMkdir(dir);
     const tenantFile = path.join(dir, 'tenant.yml');
 
-    const localAssets = {
-      tenant: {
-        friendly_name: '##ENV## Tenant',
-        enabled_locales: '@@LANGUAGES@@',
-      },
-      connections: [
-        {
-          name: 'connection-1',
-          strategy: 'waad',
-          options: {
-            tenant_domain: '##DOMAIN##',
-          },
-        },
-      ],
-    };
-
-    fs.writeFileSync(tenantFile, jsYaml.dump(localAssets));
+    const localAssets = `
+    tenant:
+      friendly_name: "##ENV## Tenant"
+      enabled_locales: @@LANGUAGES@@
+    connections:
+      - name: connection-1
+        strategy: waad
+        options:
+          tenant_domain: "##DOMAIN##"
+    `;
+
+    fs.writeFileSync(tenantFile, localAssets);
```

Comment:
Not possible to test this scenario with JSON, needs to be converted to YAML. Note the unwrapped `@@LANGUAGES@@` array replace instance.